biojava-structure/src/main/java/org/biojava/nbio/structure/Model.java

```diff
@@ -71,7 +71,7 @@ public void addChain(Chain c) {
         EntityInfo info = c.getEntityInfo();
         
         if ( info == null || info.getType() == null) {
-        	logger.warn("No entity info could be found while adding chain with asym id {} (author id {}). Will consider it a polymer chain.", c.getId(), c.getName());
+        	logger.debug("No entity info could be found while adding chain with asym id {} (author id {}). Will consider it a polymer chain.", c.getId(), c.getName());
```

Comment:
I'd keep this at least on INFO level, this is quite a strong indication that there's something wrong in reading the structure.